shortcodes.php

```diff
@@ -1514,19 +1514,23 @@ function sc_expert_photos($atts = Array())
  * @return string.
  * @author Cadie Brown
  **/
-function sc_feature_post_meta($atts = Array())
+function sc_feature_post_meta( $atts = Array() )
 {
 	global $post;
 
-	$css = (isset($atts['css'])) ? $atts['css'] : '';
+	$css = ( isset( $atts['css'] ) ) ? $atts['css'] : '';
 
-	$byline = get_post_meta($post->ID, 'author_byline', True);
-	$byline = ($byline != '') ? $byline : get_the_author();
+	$byline = get_post_meta( $post->ID, 'author_byline', True );
+	$byline = ( $byline != '' ) ? $byline : get_the_author();
+	$updated_date = get_post_meta( $post->ID, 'updated_date', True );
 
 	ob_start(); ?>
 	<div class="<?php echo $css; ?>" id="meta">
 		<div>
-			<p id="byline-date">By <?php echo $byline; ?> <span class="hidden-mobile">|</span><br class="visible-mobile"> <?php echo date('F j, Y', strtotime($post->post_date)); ?></p>
+			<p id="byline-date">By <?php echo $byline; ?> <span class="hidden-mobile">|</span><br class="visible-mobile"> <?php echo date( 'F j, Y', strtotime( $post->post_date ) ); ?></p>
+			<?php if( $updated_date ) : ?>
+				<p>Updated <?php echo date( 'F j, Y', strtotime( $updated_date ) ); ?></p>
```

Comment:
We might want to style this text to be uppercase so that it matches the published date:
https://github.com/UCF/Today-Bootstrap/blob/rc-v2.9.0/src/scss/pages/_featured-story.scss#L46

Depending on how fancy we want to get with it too we could wrap the 'Updated' text in a span and make that the Athena accessible-over-white color red, and either just have that underneath the date or floated to the right:
![screen shot 2018-11-08 at 1 46 09 pm](https://user-images.githubusercontent.com/21267784/48220583-b69ce780-e35d-11e8-9c98-c34949ab10b6.png)

Making it float to the right of the published date might be a little tricky to correct that in mobile though, since we're dealing with Bootstrap 2 in this theme.